### PR TITLE
Secure URLs

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

**Comment by vszakats** [comment]
Any issues to address with this PR to have it merged?

File: src/neon.mk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file is part of MXE. See LICENSE.md for licensing information.
 
 PKG             := neon
-$(PKG)_WEBSITE  := http://webdav.org/neon/
+$(PKG)_WEBSITE  := https://web.archive.org/web/20070925040920/www.webdav.org/neon/
 $(PKG)_DESCR    := HTTP and WebDAV client library (libneon)
```
Comment (tonytheodore) [inline src/neon.mk]: Why not use the most recent capture?

Comment (vszakats) [reply src/neon.mk]: Good point, can't remember what was the reason in this case (I took this change from an earlier patch to another project). Generally I also prefer the default/latest capture, but it's often not what's expected. In this case it is, so an update is commited.

Comment (vszakats) [reply src/neon.mk]: (and squashed)

**Comment by tonytheodore** [comment]
Thanks!

**Pull request merged**